docs/howto.rst

```diff
@@ -22,11 +22,11 @@ In order to follow the tutorial, you need a very quick setup: download or clone
 
     For instance, if you are planning to process AcqKnowledge files, install the interface dependencies as described `here <installation.html#>`_.
 
-For the tutorial, we will assume the repository was downloaded in ``/home/arthurdent/git``. Let's get there right now:
+For the tutorial, we will assume the repository was downloaded in ``/home/arthurdent``. Let's get there right now:
 
 .. code-block:: shell
 
-    cd /home/arthurdent/git/
+    cd /home/arthurdent
 
 What is in the tutorial text file?
 ##################################
```

Comment:
I also saw now that line 34 has to be adapted.

@RayStick apologies if I'm missing other points, maybe you could have a broader check when you do your review? I'll try to check the full page again.